app.html

```diff
@@ -262,8 +262,8 @@ <h2>Skills</h2>
     </article>
     <form action="https://formspree.io/f/xwkzpleb" method="post" class="contact_form" id="form">
       <fieldset class="form_fieldset">
-        <input type="text" value="" name="user_name" placeholder="Full name" maxlength="30" id="form_name" required>
-        <input type="email" value="" name="user_email" placeholder="Email address" maxlength="30" id="form_email" class="" required>
+        <input type="text" value="" id="form_name" name="user_name" placeholder="Full name" maxlength="30"  required>
+        <input type="email" value="" name="user_email" placeholder="Email address" maxlength="30" id="form_email" required>
         <textarea name="message" id="form_message" placeholder="Enter text here" cols="30" rows="10" maxlength="500" required></textarea>
```

Comment:
- [ ] It would be better if the data in the text area is stored in local storage even after page refresh. Currently, when I input values like, name, email, description, when I refresh the page, the data in the name and email fields remain the same unchanged except for the description which changes to "undefined". Kindly find the screencast below to have a fair idea
[Screencast from 2022-09-08 12:08:02 EW.webm](https://user-images.githubusercontent.com/46388113/189121181-3615814e-e9c5-41f5-8804-6a8fce6dab8c.webm)